pom.xml

```diff
@@ -100,8 +100,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.1</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>1.7</source>
```

Comment:
I think it would be perfect if you add a comment here to tell the following partners why this setting is necessary: the JAVA version of the environment where the client is located may be different from the server, tubemq-client, tubemq-core should maintain the 1.7 version.

Reply:
And this pr need a jira number ，like [TubeMQ-xxx] xxxxx

Reply:
Okay, I will add a comment here